mosqlient/datastore/infodengue.py

```diff
@@ -0,0 +1,43 @@
+from datetime import date
+from typing import Optional, Literal
+
+from pydantic import BaseModel
+
+from mosqlient import types
+from mosqlient.requests import get_all_sync
+from mosqlient._utils import parse_params
+
+
+class Infodengue(BaseModel):
+    @classmethod
+    def get(
+        cls,
+        disease: Literal["dengue", "zika", "chikungunya"],
```

Comment:
Should `disease`, `start`, `end` parameters be obligatory to get Infodengue data?